assembly/src/parsers/io_ops/mem_ops.rs

```diff
@@ -82,6 +82,22 @@ pub fn parse_mem_write(
     Ok(())
 }
 
+/// Appends operations to the span block to execute `mem_stream` instruction. The sequence of
+/// operations is: MSTREAM RPPERM.
+///
+/// This instruction requires 2 VM cycles to execute.
```

Comment:
nit: maybe this doc comment should include an `Errors` section

Reply:
Added!